srcpkgs/python-xcffib/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-xcffib'
+pkgname=python-xcffib
+version=0.8.1
+revision=1
+archs=noarch
```

Comment:
Please mark nocross instead. I do not have hardware to check, but as x86_64 and armv6l are exactly same, I guess it is because build do not even know about target arch rather than it is not dependent on arch.